backend/app/views/spree/admin/variants/_form.html.erb

```diff
@@ -17,28 +17,22 @@
       </div>
     </div>
   </fieldset>
-  <% if f.object.new_record? %>
-    <fieldset class="no-border-top no-border-bottom">
-      <div class="row">
-      <% @product.option_types.each_with_index do |option_type, index| %>
-          <div class="col-3">
-            <div class="field" data-hook="presentation">
-              <%= label :new_variant, option_type.presentation %>
-              <%= f.collection_select 'option_value_ids',
-                                      option_type.option_values,
-                                      :id,
-                                      :presentation,
-                                      { include_blank: true },
-                                      {
-                                        name: 'variant[option_value_ids][]',
-                                        class: "select2 fullwidth"
-                                      } %>
-            </div>
-          </div>
-      <% end %>
+  <fieldset class="no-border-top no-border-bottom">
+    <% @product.option_types.each_with_index do |option_type, index| %>
+      <div class="field four columns <%= 'alpha' if index % 3 == 0 %><%= 'omega' if index % 3 == 2 %>" data-hook="presentation">
```

Comment:
This goes back to using the skeleton classes. Can this be updated to use the bootstrap classes (as the removed code did)